src/NServiceBus.Core/BusConfiguration.cs

```diff
@@ -42,14 +42,15 @@ public BusConfiguration()
             Settings.SetDefault("Transactions.DefaultTimeout", TransactionManager.DefaultTimeout);
             Settings.SetDefault("Transactions.SuppressDistributedTransactions", false);
             Settings.SetDefault("Transactions.DoNotWrapHandlersExecutionInATransactionScope", false);
+
+            conventionsBuilder = new ConventionsBuilder(Settings);
         }
 
         /// <summary>
-        ///     Access to the pipeline configuration
+        ///     Access to the receive pipeline configuration
```

Comment:
Good catch, it's actually all pipelines. @SzymonPobiega is planning to fix this I think. Comment updated